mgmtd/mgmt_txn.c

```diff
@@ -2339,33 +2341,56 @@ int mgmt_txn_send_get_req(uint64_t txn_id, uint64_t req_id,
  * has registered operational state that matches the given `xpath`
  */
 int mgmt_txn_send_get_tree_oper(uint64_t txn_id, uint64_t req_id,
-				uint64_t clients, LYD_FORMAT result_type,
-				uint8_t flags, bool simple_xpath,
-				const char *xpath)
+				uint64_t clients, Mgmtd__DatastoreId ds_id,
+				LYD_FORMAT result_type, uint8_t flags,
+				bool simple_xpath, const char *xpath)
 {
 	struct mgmt_msg_get_tree *msg;
 	struct mgmt_txn_ctx *txn;
 	struct mgmt_txn_req *txn_req;
 	struct txn_req_get_tree *get_tree;
 	enum mgmt_be_client_id id;
 	ssize_t slen = strlen(xpath);
+	uint32_t wd_options;
 	int ret;
 
 	txn = mgmt_txn_id2ctx(txn_id);
 	if (!txn)
 		return -1;
 
+	switch (flags & GET_DATA_FLAG_WD_MASK) {
```

Comment:
Again unnecessary code, just use the libyang type.

Reply:
As discussed above, we're using own values for the `defaults` parameter, so the translation is needed.